src/messaging/src/v1/fcm_service_client.ts

```diff
@@ -0,0 +1,362 @@
+/* eslint-disable */
```

Comment:
Let's put these in a directory named `generated` instead of `src`. The full path should be something like `src/messaging/generated/v1/fcm_service_client.ts`.

Also, how do you feel about keeping the generated sources and protos in completely separate directory hierarchy. Something like:

```
REPO_ROOT
  src/
    messaging/
  generated/
    messaging/
      src/
        v1/
      protos/
```

This will likely require some work at the tsconfig and our build config (gulpfile) to compile and bundle correctly.

Reply:
I agree that it will probably be a bit of extra work but think it'll be clearer/easier to parse, so definitely worth it.

Reply:
Do you want to do that as part of this PR, or do that later? I think for our immediate purposes what you have here now is good enough.

Reply:
It's a quick/easy enough change, and I think it's worth not having to do a larger refactor down the line.